### PR TITLE
Closes #66

### LINKED ISSUE #66: General Issue: STEP 5 ADIS
### Background Information

Creation of PARCAT1, SERCAT1/N and CUTOFF0x variables.

### Definition of Done

_No response_

### PR DESCRIPTION
**Pull request by federicobaratin**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/main/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/main/articles/unit_test_guidance.html#writing-unit-tests-in-admiral)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page.
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging.
- [x] Address all merge conflicts and resolve appropriately.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by federicobaratin** [comment]
Hi @bms63 !
Sorry to bothering you, but could you please help me in this? :)
Thanks!

**Comment by federicobaratin** [comment]
Hi!
@bms63 I have checked the errors.
The first one is related to some mispelled terms (i think in Pfizer code, but not sure).
The second one is due to the fact that the PARAM functions do not find ISSTRESN variable (that should be present).
Do I need to change something in my code?

**Comment by federicobaratin** [comment]
Same errors like in 65 pull request

File: inst/templates/ad_adis.R [hunk 1]
```diff
@@ -145,3 +169,136 @@ is4 <- derive_vars_merged_lookup(
...
+    AVAL = case_when(
+      # ISORRES values without > or <
+      DERIVED == "ORIG" & !is.na(ISSTRESN) & ISSTRESN < ISLLOQ ~ ISLLOQ / 2,
+      DERIVED == "ORIG" & !is.na(ISSTRESN) & ISSTRESN >= ISLLOQ & ISSTRESN < ULLOQ ~ ISSTRESN,
+      DERIVED == "ORIG" & !is.na(ISSTRESN) & ISSTRESN >= ULLOQ ~ ULLOQ,
+      DERIVED == "LOG10" & !is.na(ISSTRESN) & ISSTRESN < ISLLOQ ~ log10(ISLLOQ / 2),
+      DERIVED == "LOG10" & !is.na(ISSTRESN) & ISSTRESN >= ISLLOQ & ISSTRESN < ULLOQ ~ log10(ISSTRESN),
+      DERIVED == "LOG10" & !is.na(ISSTRESN) & ISSTRESN >= ULLOQ ~ log10(ULLOQ),
+      DERIVED == "4FOLD" & !is.na(ISSTRESN) & ISSTRESN < ISLLOQ ~ ISLLOQ,
+      DERIVED == "4FOLD" & !is.na(ISSTRESN) & ISSTRESN >= ISLLOQ & ISSTRESN < ULLOQ ~ ISSTRESN,
```
Comment (bms63) [inline inst/templates/ad_adis.R:197]: @federicobaratin I found some use of `!!` in this `case_When`. I don't think this was intentional and removed from the code.

**Comment by bms63** [comment]
@pharmaverse/admiralvaccines yay!  This one has all passed!! 

You can click on the committ history to see what @federicobaratin and I did to fix the issues - all very simple 
image

File: inst/templates/ad_adis.R (hunk 1 above)
Comment (federicobaratin) [reply inst/templates/ad_adis.R]: Oh!
No, not intentional at all.
Thanks Ben!

**Pull request merged**